pkg/api/ecosystem/blueprintRestClient.go

```diff
@@ -0,0 +1,242 @@
+package ecosystem
+
+import (
+	"context"
+	"time"
+
+	v1 "github.com/cloudogu/k8s-blueprint-operator/pkg/api/v1"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/retry"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	appLabelKey      = "app"
+	appLabelValueCes = "ces"
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>const `appLabelValueCes` is unused